wal-routing/src/router/builder.rs

```diff
@@ -0,0 +1,166 @@
+use super::{PageRenderer, Router};
+use std::{collections::HashMap, marker::PhantomData};
+use wal::component::{node::AnyComponentNode, root::RootComponent};
+
+const ERROR_PATH: &'static str = "/404";
```

Comment:
to bardziej not found niz error pewnie

Reply:
i consty z defaultu są statyczne z tego co wiem

Reply:
+1